generator/AWSPSGeneratorLib/HelpMaterials/Examples/LM/Get-LMFunctionList.xml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>Get-LMFunctionList | Sort-Object -Property CodeSize | Select-Object FunctionName, RunTime, Timeout  
```

Comment:
It would be nice to add CodeSize as well in the output

Reply:
Done. Made the changes